users/admin.py

```diff
@@ -19,10 +21,28 @@ class Meta(UserChangeForm.Meta):
 
 
 class JustSpacesUserAdmin(UserAdmin):
-    add_fieldsets = UserAdmin.add_fieldsets + (
-        (None, {
-            'fields': ('agency',),
-        }),
+    permissions_description = 'Just Spaces has three levels of user \
+        permissions:<br /><br /> \
+        1. <b>Field users</b> can run published surveys.<br /> \
+        2. <b>Staff users</b> have all the permissions of field users. They can \
+            also create, edit, and delete Agencies, Studies, Study Areas, \
+            Surveys, and Locations. They can publish surveys so they can \
+            be run by field users. Staff users can also view all collected \
+            data and design data visualizations on the collected data pages.<br /> \
+        3. <b>Superusers</b> have all the permissions of staff users. They can \
+            also create, edit, and delete other users. Only superusers can set \
+            and change user passwords. <b>All superusers should \
+            also have "Staff status" selected below</b>.<br />'
```

Comment:
This copy is super helpful! I think one way we could make it even better would be to explain specifically which buttons need to be checked for each user type, similar to what you've hinted at with superusers. Here's how I would phrase it:

> This app employs three classes of users:
>
> 1. `Field users` can run published surveys. **To create a field user, check 'Active' below.**
> 2. `Staff users` have all the permissions of field users. They can also create, edit, and delete Agencies, Studies, Study Areas, Surveys, and Locations. They can publish surveys so they can be run by field users. Staff users can also view all collected data and design data visualizations on the collected data pages. **To create a staff user, check 'Active' and 'Staff status' below.**
> 3. `Superusers` have all the permissions of staff users. They can also create, edit, and delete other users. Only superusers can set and change user passwords. **To create a superuser, check all the boxes below.**

Reply:
Looks great! I'll modify and then merge this.